protob/messages.proto

```diff
@@ -913,6 +922,57 @@ message CosiSignature {
 	optional bytes signature = 1;		// Signature
 }
 
+//////////////////////
+// Stellar messages //
+//////////////////////
+
+/**
+ * In: request for the public key at the specified index
+ */
+message StellarGetPublicKey {
+	required uint32 index = 1;			// Account index (derived as m/44'/148'/index')
```

Comment:
Use `optional` instead of `required ` https://developers.google.com/protocol-buffers/docs/proto

Reply:
Can you clarify why this one should be optional? I'd like to ensure that an index is always passed with this message.